restructuredtext_lint/test/test.py

```diff
@@ -129,9 +129,28 @@ def test_rst_lint_many_files(self):
             # python ../cli.py test_files/valid.rst invalid.rst
             subprocess.check_output((sys.executable, rst_lint_path, valid_rst, invalid_rst))
         output = str(e.exception.output)
-        # 'rst-lint' should exit with error code 1:
-        self.assertEqual(e.exception.returncode, 1)
+        # 'rst-lint' should exit with error code 2 as linting failed:
+        self.assertEqual(e.exception.returncode, 2)
         # There should be a least one valid .rst file:
         self.assertIn('is clean', output)
         # There should be a least one invalid rst file:
         self.assertIn('WARNING', output)
+
+    def test_fail_low(self):
```

Comment:
Since these two tests are related, we should prob use similar titles. Can we rename them to `test_level_fail` and `test_level_pass`?